### PR TITLE
55 GitHub setup project source tree

### LINKED ISSUE #55: Github - Setup project source tree
Prepare the github initial structure for the following:

1. Frontend - Primarily javascript (source, tests, and build scripts; assuming vue.js for now)
2. Backend - Primarily python (shared 'shortlist' package, django core, tests, etc.)
3. Continuous Integration - travisci integration
4. API Testing - postman (other?) API testing
5. Monitoring - new relic, AWS logging, or datadog (TBD)
6. Docs - team-specific documentation on how to perform certain tasks across this project's stack
7. Deployment - tools for deployment on local machines (dev), test environment (aws, or other), and prod environment (aws or other)

Additionally, use this as an opportunity to demonstrate how to create a branch and how to initiate/accept a pull-request (whole team as reviewers)

### PR DESCRIPTION
**Pull request by waltage**
Contributing a high-level tree structure proposal as we begin the architecture discussions and the design docs;  

I'm also taking this opportunity to demonstrate some of the functionality we'll use over time -- 

1. Taking an issue and creating a separate "branch" for it (this case it's 55-github-setup....)
2. Making changes and commits in a local environment (with action phrases as the commit message, and as individually reversible changes)
3. Initiating a pull request (PR) for reviewers to double check (in this case I've selected the whole team)
4. Once **all reviewers** give a "Looks Good To Me" or (LGTM), then we can "merge" the PR into the corresponding main branch (in this case, it will go into "develop" while we wait for the rest of the branch details to come from Prof & TA)
5. Until we have all approval from reviewers, we need to make changes and collaborate on getting it to LGTM status, or to close it out 

Our team will make our own decisions about how our review process will work, but the above is an "average" flow that I've illustrated here in the meantime.

### CODE REVIEW

**Review by yalexublue — APPROVED** [review]
Looks good! Some folder name terms I don't really know what they refer to but I'm sure they have their meanings. I agree to merge to Develop

**Comment by waltage** [comment]
> Looks good! Some folder name terms I don't really know what they refer to but I'm sure they have their meanings. I agree to merge to Develop

For practice, revoke your approval and go into the changes to find a folder you don't understand yet, and make a comment/question on it @yalexublue

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Documentation for Team 2
+
+1. Instructions for AWS 
+2. Instructions for GitHub (commits, branches, pull requests, setup, etc.)
+3. Instructions for Local development
+4. Instructions for Travis (continuous integration)
+5. Instructions for Postman (testing the API)
```
Comment (waltage) [inline docs/README.md:7]: What is postman?

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Documentation for Team 2
+
+1. Instructions for AWS 
+2. Instructions for GitHub (commits, branches, pull requests, setup, etc.)
+3. Instructions for Local development
+4. Instructions for Travis (continuous integration)
```
Comment (jainsvaibhav) [inline docs/README.md:6]: test comment

**Review by jainsvaibhav — CHANGES_REQUESTED** [review]
please make change

**Review by adelaidemz — APPROVED** [review]

**Review by jainsvaibhav — APPROVED** [review]
LGTM

**Pull request merged**